node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/text/SnippetGenerator.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.node.admin.task.util.text;
+
+/**
+ * @author hakon
+ */
+public class SnippetGenerator {
+
+    private static final int ASSUMED_OMIT_TEXT_LENGTH = omitText(1234).length();
+
+    private static String omitText(int omittedLength) { return "[..." + omittedLength + " chars omitted]"; }
+
+    /** Returns a snippet of approximate size. */
+    public String makeSnippet(String text, int sizeHint) {
+        if (text.length() < sizeHint) return text;
```

Comment:
This should be `<=`

Reply:
Also you're better off returning `text` if it smaller than omit text length.

Reply:
Done

By "omit text length" I take it you mean ASSUMED_OMIT_TEXT_LENGTH and not omitText(text.length).length(). Done.